src/platforms/rust/guides/actix-web/index.mdx

```diff
@@ -9,12 +9,15 @@ The `sentry-actix` crate adds a middleware for [`actix-web`](https://actix.rs/)
 
 To use this middleware, configure Sentry then add it to your actix web app as a middleware. Because actix is generally working with non sendable objects and is highly concurrent, this middleware creates a new hub per request. As a result, many of the Sentry integrations, such as breadcrumbs, do not work unless you bind the actix hub.
 
+NB: Macros like `#[tokio::main]` and `#[actix_web::main]` are not supported. The Sentry client must be initialized before the async runtime is started so that all threads are correctly connected to the Hub.
```

Comment:
Are you using NB as in nota bene here? We typically avoid latin phrases/acronyms in docs as they don't translate well for non-native speakers. Maybe you can use a note here instead?

Reply:
Yeah, that's what I was going for. I'll change it.